### DISCUSSION TITLE
build language chinese ?

### DISCUSSION THREAD

**Discussion by Ruzikejiao999** [statiqdev/Discussions#37] [Q&A]
how to build output html files .display well with chinese language

**Comment by daveaglick** [comment]
Statiq will happily generate HTML files from whatever Markdown, Razor, etc. it finds in both the `input` and `theme/input` folders. In that sense it's language-agnostic. If the Markdown contains Chinese, the output will as well. That said, I don't speak or write anything but English so templates I create will likely be English-based. It would be awesome if the community eventually comes up with templates and themes in other languages, but that's not something I'll be able to do directly.

**Reply by CrazyKodo** [reply]
I guess he means
<img width="1235" height="825" alt="image" src="https://github.com/user-attachments/assets/9d0261aa-3f06-48da-b1f4-7d78d9838951" />
The Chinese characters become �Ʒ���


Sorry...ignore this..... It's working fine after I save my Razer file with UTF-8.
